### PR TITLE
JOINDIN-651 Back populate new table

### PR DESCRIPTION
**Pull request by exussum12**
Probably need a discussion on how to backpopulate the internal links.

Speaker + talk title I think would be a match ? Any other suggestions ?

### CODE REVIEW

**Comment by railto** [comment]
ok to test

**Comment by railto** [comment]
This can not be merged until #392 is first merged, note the patch number will need to be increased to 71

**Comment by exussum12** [comment]
@liam-wiltshire just a suggestion with these patch numbers, can they be ticket numbers until they are merged ? They only mean something when they are merged and its impossible to know before hand which numbers will be free

**Comment by liam-wiltshire** [comment]
You ok to update this to 71 please? :)

File: db/patch71.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+INSERT INTO `joindin`.`talk_links` (`talk_id`, `talk_type`, `url`)
+SELECT
+  id,
+  1,
+  slides_link
+FROM
+  talks
+WHERE slides_link <> ''
```
Comment (liam-wiltshire) [inline db/patch71.sql]: This is currently throwing a syntax error in mySQL - think it's just a missing semi-colon however.

Comment (exussum12) [inline db/patch71.sql]: Your right. Will fix now sorry

**Pull request merged**